lib/jekyll_pages_api/page.rb

```diff
@@ -3,14 +3,17 @@
 module JekyllPagesApi
   # wrapper for a Jekyll::Page
   class Page
+    HTML_EXTS = %w(.html .md .markdown .textile)
     attr_reader :page
 
-    def initialize(page)
+    def initialize(page, site)
       @page = page
+      @site = site
     end
 
     def html?
-      %w(.html .md).include?(self.page.ext)
+      path = @page.path
```

Comment:
So I figure [this is the rationale for `self.page` vs. `@page`]? Done, but I fundamentally object to the premise that the `self.page` form is preferable because it encourages subclassing. ;-)